service.h

```diff
@@ -0,0 +1,44 @@
+#include <memory>
+#include <string>
+#include <vector>
+#include <thread>
+#include <mutex>
+
+#include <grpcpp/channel.h>
+
+#include "backend_client.h"
+#include "service.grpc.pb.h"
+#include "data_storage_types.grpc.pb.h"
+
+// A backend service to receive requests from client (command line)
+class Service final : public chirp::ServiceLayer::Service {
+ public:
+   Service() : backend_client_(grpc::CreateChannel("localhost:50002", grpc::InsecureChannelCredentials())) {}
+  
```

Comment:
Is the space between function declarations necessary? It's not consistent with other .h files.